rss/serializers/atom_1_0.py

```diff
@@ -1,15 +1,81 @@
 """Serializer for Atom 1.0"""
 
-from flask.json import jsonify
+from feedgen.feed import FeedGenerator
+from flask import url_for
 from typing import Tuple, Optional
 from arxiv import status
 from rss.serializers.serializer import Serializer
+from rss.serializers.atom_extensions import ArxivExtension, ArxivEntryExtension
 from elasticsearch_dsl.response import *
 
+import datetime
+import pytz
+
 
 class Atom_1_0(Serializer):
 
     def get_xml(self, response: Response) -> Tuple[Optional[dict], int]:
-        data = jsonify({'version': "atom 1.0"})
+
+        fg = FeedGenerator()
+        fg.register_extension("arxiv", ArxivExtension, ArxivEntryExtension, rss=False)
+        fg.id("http://arxiv.org/rss/version=atom_1.0")
+        archive = response.hits[0]["primary_classification"]["archive"]
+        fg.title(archive["id"] + " updates on arXiv.org")
+        fg.link(href='http://arxiv.org/rss/version=atom_1.0', rel='self', type='application/atom+xml')
+        fg.updated(datetime.datetime.utcnow().replace(tzinfo=pytz.utc))
+
+        # TODO - Try to remove generator element?  This doesn't work - code ignores "None"
+        # fg.generator(None)
```

Comment:
I noticed in `rss/serializers/rss_2_0.py` , you have a slightly different syntax that isn't being commented out: `feed.generator = None`. While I guess doing the same thing here probably doesn't work, would be interesting to know. On the other hand, I can't think of why it would be harmful to have the [generator element](https://www.w3schools.com/xml/rss_tag_generator.asp) - I would either put it down as `arxiv.org` or `arXiv-NG` (in all cases, not just this atom generator, if we decide to do it. CC @mhl10 & @erickpeirson

Reply:
In both cases, the generator element is set to the name/info for the project I am calling.  In the case of the RSS implementation, Eric said we'd like to remove that element and I was able to do so in the code.  A similar approach for Atom (calling a different library) did not work because the library creates the default value when you pass in "None" to the method.

We could get the code and modify it to not output this element at all.  I don't think we can override our own version of the generator property in our extension of their model such that an "arXiv" generator will be specified.  I wasn't sure if it was OK to claim we generated the content when we are calling their library.  At the moment, the element is produced with Feed Generator in the generator element.